tests/acceptance/setup.js

```diff
@@ -110,6 +113,10 @@ After(function closeSessionForEnv () {
   return closeSession()
 })
 
+After(function rollbackConfigsOnLocal () {
+  return rollbackConfigs(client.globals.backend_url)
```

Comment:
@jasson99  skip this part when running on ocis

Reply:
done